modules/local/download_proteins.nf

```diff
@@ -0,0 +1,43 @@
+process DOWNLOAD_PROTEINS {
+    tag "$microbiome_ids"
+    label 'process_low'
+
+    conda (params.enable_conda ? "conda-forge::biopython=1.78" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/biopython:1.78' :
+        'quay.io/biocontainers/biopython:1.78' }"
+
+    input:
+    val    microbiome_ids
+    path   microbiome_files
```

Comment:
Just to make sure, these are not sample related, i.e. not rather a tuple?

Reply:
No, all the ids and files are collected and given to this process.